interaction_lib.py

```
@@ -0,0 +1,307 @@
+    Returns:
+        str: HTML table
+    """
+    from logic import fmt_p_with_styling, COLORS
+    
+    if not results:
+        return "<p class='text-muted'>No interaction results available.</p>"
+    
+    effect_col = "OR" if model_type == 'logit' else "IRR"
+    
+    rows = []
+    for int_name, res in results.items():
```

Ruff (0.14.10)

Loop control variable `int_name` not used within loop body

Rename unused `int_name` to `_int_name`

(B007)